pykg2tbl/j2/jinja_sparql_builder.py

```diff
@@ -3,10 +3,14 @@
 
 from jinja2 import Environment, FileSystemLoader, meta
 
-from pykg2tbl.service import SparqlBuilder
+from pykg2tbl.query import SparqlBuilder
 
 log = logging.getLogger(__name__)
 
+TEMPLATES_FOLDER = os.path.join(
```

Comment:
rename to `DEFAULT_TEMPLATES_FOLDER` as it should be able to override and add another